mantle/kola/tests/upgrade/basic.go

```diff
@@ -220,6 +227,40 @@ func (g *Graph) addUpdate(c cluster.TestCluster, m platform.Machine, version, pa
 	g.sync(c, m)
 }
 
+// XXX: consider making this distinction part of FCOS itself?
```

Comment:
Right like hmm...part of the ostree commit object?

Reply:
Yeah, that's the obvious place to put it. My only hesitation is that it technically makes `fedora-coreos.stream` less authoritative because FCOS nodes already do have that information by checking that the stream is one of `stable`, `testing`, or `next`.

Maybe a cleaner approach is to ship a definition file where we say what the production streams are, and any software can just check whether the value of `fedora-core.stream` is in that list.

Reply:
@jlebon the latter goes in the direction of https://github.com/coreos/fedora-coreos-tracker/issues/317.